### ISSUE TITLE
18287 - Configure WebDAV Access for your NGINX server - ALL DISTROS.html

### ISSUE THREAD

**Issue by randomcode17** [gnakan/gd-cloud-docs#72]
18287 - Configure WebDAV Access for your NGINX server - ALL DISTROS

**Comment by randomcode17** [comment]
Please test this article on all distros.

**Comment by susankayjordan** [comment]
Errors fixed ready for Editing

**Comment by susankayjordan** [comment]
@Carla-Johnson  This article seems to be missing parts it may need to go back to the content creator

**Comment by randomcode17** [comment]
From my perspective, the article is complete.  What’s missing?

**Comment by susankayjordan** [comment]
There was a bunch of the "template text" in the middle of the article  (about 30 lines or so and than the last couple of lines, I was not sure if  something got missed.

**Comment by randomcode17** [comment]
Where are you pulling the file from.  I saved it to the github repo.

**Comment by Carla-Johnson** [comment]
I just moved it to the Editing column, but when I went to open it in the folder on my desktop, it was not there. Gary, any ideas?

-Carla

**Comment by susankayjordan** [comment]
@randomcode17  I pulled the file from GIT Hub.... I thought it was really weird that your file would have that, your stuff usually is pretty clean.

**Comment by randomcode17** [comment]
Is it still looking half-finished?  It might have been imported from the SP repo in its half-baked form.  Can you try doing a “get – latest” (or the equivalent) and see if it still looks messed up?  My version is completely clean.

**Comment by Carla-Johnson** [comment]
Yes. It doesn’t have a proper title, intro, or description. Should I just send it back to you?

-Carla

**Comment by Carla-Johnson** [comment]
Sending back.

**Comment by brian-r-miller** [comment]
Hang on – I have a sync issue that is causing this mess.

**Comment by gnakan** [comment]
@brian-r-miller @Carla-Johnson 

So the issue appears to be in SP. 18287 exists as two different files, one for All Distros and one for Ubuntu/Debian. The content from the Ubuntu/Debian is valid, just needs to be in the All Distros file.

My recommendation: Pull the content from the Ubuntu/Debian version, replace the content of the file in github and sync.

**Comment by susankayjordan** [comment]
Coding Complete Validated

**Comment by Carla-Johnson** [comment]
Edited

**Comment by sloisellegd** [comment]
Edit doc: https://docs.google.com/document/d/1q5nyiKuzXYHzbjaHa5mkDCyYUx2PqFf_JDHajimJM_A/edit
